src/common/mmo.h

```diff
@@ -1291,15 +1304,39 @@ enum e_class {
 	JOB_BABY_RANGER2 = 4111,
 	JOB_BABY_MECHANIC2 = 4112,
 
+	JOB_THIRD_JOB_LAST = 4113,
+
 	JOB_SUPER_NOVICE_E = 4190,
 	JOB_SUPER_BABY_E = 4191,
 
 	JOB_KAGEROU = 4211,
 	JOB_OBORO = 4212,
+
 	JOB_REBELLION = 4215,
 
 	JOB_SUMMONER = 4218,
 
+	JOB_BABY_SUMMONER = 4220,
+	JOB_BABY_NINJA = 4222,
+	JOB_BABY_KAGEROU = 4223,
+	JOB_BABY_OBORO = 4224,
+	JOB_BABY_TAEKWON = 4225,
+	JOB_BABY_STAR_GLADIATOR = 4226,
+	JOB_BABY_SOUL_LINKER = 4227,
+	JOB_BABY_GUNSLINGER = 4228,
+
+	JOB_BABY_REBELLION = 4229,
+
+	JOB_BABY_STAR_GLADIATOR2 = 4238,
+
+	JOB_STAR_EMPEROR = 4239,
+	JOB_SOUL_REAPER = 4240,
+
+	JOB_BADY_STAR_EMPEROR = 4241,
```

Comment:
`BABY`

Reply:
this changed in other commits to other list